ParisBNF/abb/BNFabb152.xml

```diff
@@ -240,93 +247,90 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   celui-ci était moine ou prêtre, comme le suggère l'appellation de « père » employée dans
                   cet acte et dans l’acte du folio 60r, pouvait-il dans le même temps être
-                  <roleName type="other">śeyum</roleName>, c’est-à-dire nommé à une charge de pouvoir, et
+                  <roleName type="title">śeyum</roleName>, c’est-à-dire nommé à une charge de pouvoir, et
                   ecclésiastique ? La question du recouvrement des charges et fonctions laïques et
                   ecclésiastiques reste à explorer.</note>
                            <note xml:id="note3-3" n="2" type="traduction">Cette église se trouve dans l'Amḫara
                   Sayent. Elle fut fondée par le roi <persName ref="PRS7481NaodA">Nā‘od</persName>
                   (1494-1508), père de Lebna Dengel. Ce dernier la fit reconstruire et lors de
                   l’inauguration du nouveau bâtiment en janvier 1521 y fit déposer les restes de son père,
-                  selon le récit de Francisco Alvarez (<bibl>Àlvares, 1558, fol. 163, 164v, 212v
-                    sqq.</bibl>) . Cf aussi une synthèse de l’histoire de l’église dans : <bibl>Derat,
-                      2007</bibl>. </note>
+                  selon le récit de Francisco Alvarez (<bibl><ptr target="bm:Alvarez1558PresterJohn"/><citedRange unit="folio">163, 164v, 212v  sqq.</citedRange></bibl>) . 
+                               Cf aussi une synthèse de l’histoire de l’église dans : <bibl><ptr target="bm:Derat2007MakanaSellase"></ptr></bibl>. </note>
                            <note xml:id="note3-4" n="3" type="traduction">Le texte ge’ez porte le chiffre 68 qui est
-                  incompatible avec ce système de notation des mois lunaires. Voir <bibl>Neugebauer, 1979,
-                    p. 139-142</bibl>. Les chiffres 20 et 60, respectivement ፳ et ፷ peuvent en effet être
+                               incompatible avec ce système de notation des mois lunaires. Voir <bibl><ptr target="bm:Neugebauer1979Astronomy"></ptr><citedRange unit="page">139-142</citedRange></bibl>. Les chiffres 20 et 60, respectivement ፳ et ፷ peuvent en effet être
                   aisément confondus, aussi rectifions-nous 68 en 28.</note>
                            <note xml:id="note3-5" n="4" type="traduction">Il s’agirait, si cette date était exacte,
                   du début du mois de février 1520. Peut-être peut-on supposer une erreur du scribe et il
                   faudrait alors lire « année de la miséricorde 173 », ce qui coïnciderait avec la
                   présence de Lebna Dengel à Makāna Śellāsē pour son inauguration à la fin janvier 1521.
                   Alvarez mentionne en effet explicitement qu’à cette occasion, le roi renouvela les
                   concessions foncières de Makāna Śellāsē (<bibl>
-                                            <ref target="http://gallica.bnf.fr/ark:/12148/bpt6k54543m.image.f361.pagination">Alvarez,
```

Comment:
this is a reference to the cited rage, I would not suppress it and keep the ref before after or within citedRange/@target. it will not be visualized but will be correctly encoded and preserved